packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -362,14 +361,10 @@ describe("editable-tree: editing", () => {
 	});
 
 	it("validates schema of values", async () => {
-		const schemaData: SchemaData = {
-			treeSchema: schemaMap,
-			globalFieldSchema: new Map([[rootFieldKey, fieldSchema(FieldKinds.value)]]),
-		};
-
-		const [, trees] = await createSharedTrees(schemaData, [
-			{ type: stringSchema.name, value: "x" },
-		]);
+		const [, trees] = await createSharedTrees(fullSchemaData, [personJsonableTree()]);
+		const person = trees[0].root as Person;
+		const nameNode = person[getField](brand("name")).getNode(0);
+		const ageNode = person[getField](brand("age")).getNode(0);
```

Comment:
Are these lines testing something? If so, should there be asserts?

Reply:
there was a merge conflict in this tests and looks like I resolved it poorly. Let me fix that.